### PR TITLE
Add Head and shadowable Layout components

### PR DESCRIPTION
**Pull request by jxnblk**
This adds two components to the theme:

- Head component for taking the ContentfulLandingPage fields and rendering head metadata for the page; this will be updated according to the changes in the content model
- A noop Layout component that wraps the landing page template and is intended to be shadowed in the users site

### CODE REVIEW

File: gatsby-theme-landing-page/gatsby-config.js [hunk 1]
```diff
@@ -2,6 +2,7 @@
 module.exports = (opts = {}) => {
   return {
     plugins: [
+      'gatsby-plugin-image',
```
Comment (jxnblk) [inline gatsby-theme-landing-page/gatsby-config.js:5]: This seems to resolve the error in the logs, but we can adjust this as needed once we get images in Contentful

File: gatsby-theme-landing-page/src/components/head.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  title,
+  description,
+}) {
+  return (
+    <Helmet
+      htmlAttributes={{
+        lang: 'en-US'
+      }}>
+      <title>{title}</title>
+      <meta name='description' content={description} />
```
Comment (jxnblk) [inline gatsby-theme-landing-page/src/components/head.js:14]: We can add OG metadata and images once we have that in Contentful

File: src/gatsby-theme-landing-page/components/layout.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import * as React from 'react'
```
Comment (jxnblk) [inline src/gatsby-theme-landing-page/components/layout.js:1]: In a real-world site, this component would probably exist somewhere else and the file here would re-export it. We can move this later to better show best practices and make the starter API simpler

File: gatsby-theme-landing-page/gatsby-config.js (hunk 1 above)
Comment (jxnblk) [reply gatsby-theme-landing-page/gatsby-config.js:5]: And it looks like the deploy failed due to the same image plugin error we were seeing locally

Comment (thinkybeast) [reply gatsby-theme-landing-page/gatsby-config.js:5]: [We need all three: ](https://www.gatsbyjs.com/plugins/gatsby-plugin-image/#installation)

```
   `gatsby-plugin-image`,
    `gatsby-plugin-sharp`,
    `gatsby-transformer-sharp`, 
```

**Review by thinkybeast — APPROVED** [review]
Ready to go, though I'd recommend fixing the [gatsby plugin image installation](https://www.gatsbyjs.com/plugins/gatsby-plugin-image/#installation) first

**Pull request merged**